src/libraries/Common/tests/System/DirectoryServices/LdapConfiguration.cs

```diff
@@ -110,7 +118,14 @@ internal static LdapConfiguration GetConfiguration(string configFile)
 
                     child = config.Element("Password");
                     if (child != null)
-                        password = child.Value;
+                    {
+                        string val = child.Value;
+                        if (val.StartsWith("%") && val.EndsWith("%"))
```

Comment:
nit: use '%' instead of "%"

Reply:
This builds against .NET Framework or I would..